src/Compilers/Core/Portable/Text/SourceText.cs

```diff
@@ -560,6 +564,27 @@ public virtual void Write(TextWriter writer, TextSpan span, CancellationToken ca
             }
         }
 
+        /// <summary>
+        /// SHA1 or SHA256 checksum (determined by <see cref="ChecksumAlgorithm"/>.  Computed using the original bytes
```

Comment:
```suggestion
        /// SHA1 or SHA256 checksum (determined by <see cref="ChecksumAlgorithm"/>).  Computed using the original bytes
```

Reply:
Perhaps, instead of "SHA1 or SHA256" I'd just say "cryptographic" to future proof the comment, in case we add SHA512 or other alg.

Reply:
note: this pr has been merged.  Feel free to send updates :)

Reply:
tweaking in https://github.com/dotnet/roslyn/pull/70838